samples/wifi/radio_test/sample_description.rst

```diff
@@ -560,6 +561,50 @@ Testing
 
          See :ref:`wifi_radio_ficr_prog_subcmds` for a list of available subcommands.
 
+         * To set regulatory domain with the following configuration:
```

Comment:
Line 562 "See :ref:`wifi_radio_ficr_prog_subcmds` for a list of available subcommands." can be moved after line 605.